packages/jsii-pacmak/bin/jsii-pacmak.ts

```diff
@@ -122,6 +122,12 @@ import { VERSION_DESC } from '../lib/version';
       // This is expected to be a path, which should be normalized
       normalize: true,
     })
+    .option('java-use-default-m2', {
+      type: 'boolean',
+      desc: 'Use default M2 repository when packaging a Java package.',
+      default: false,
```

Comment:
I have set the default to `false` for this setting to maintain backward compatibility. I am wandering whether this should be defaulted to `true` in future as I am not sure what negative side effects there would be by using the user's .m2 repository.